.gitignore

```diff
@@ -350,3 +350,4 @@ MigrationBackup/
 .ionide/
 
 .idea/
+*.editorconfig

```

Comment:
Editor config should be a part of solution, because it defines project-wide code-style settings.

Reply:
Removed from .gitignore